src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -380,6 +380,7 @@ def evaluate_from_recipe(cls, cfg: DictConfig) -> Tuple[nn.Module, Tuple]:
             pretrained_weights=cfg.checkpoint_params.pretrained_weights,
             checkpoint_path=cfg.checkpoint_params.checkpoint_path,
             load_backbone=cfg.checkpoint_params.load_backbone,
+            checkpoint_num_classes=cfg.checkpoint_params.checkpoint_num_classes,
```

Comment:
I know this would work on our recipes, but what if someone uses an older recipe that doesnt inherit from `default_checkpoint_params` ?
I think this would break, and it that case we should use `get_params` for more safety.

Reply:
Fair point!

Reply:
Fixed in 71ab980650db223e428f8cf91a07d5d1a645502d